src/sentry/auth/helper.py

```diff
@@ -849,3 +856,26 @@ def bind_state(self, key, value):
 
     def fetch_state(self, key=None):
         return self.state.data if key is None else self.state.data.get(key)
+
+    def disable_2fa_required(self):
+        require_2fa = self.organization.flags.require_2fa
+        if require_2fa and require_2fa.is_set:
+            self.organization.update(
+                flags=F('flags').bitand(~Organization.flags.require_2fa)
+            )
```

Comment:
Should you return here if `require_2fa` is not set? Right now it looks like the code will emit a log message and create an audit entry in all cases.